### PR TITLE
Bug 1747950: MachineSet and Machine tab should not select namespace

### PR DESCRIPTION
**Pull request by jhadvig**
/assign @spadgett @dtaylor113 

https://jira.coreos.com/browse/CONSOLE-1562

### CODE REVIEW

File: frontend/public/components/nav/admin-nav.tsx [hunk 1]
```diff
@@ -146,9 +148,18 @@ const AdminNav = () => (
...
-      <ResourceNSLink resource={referenceForModel(MachineSetModel)} name="Machine Sets" required={FLAGS.CLUSTER_API} />
-      <ResourceNSLink resource={referenceForModel(MachineAutoscalerModel)} name="Machine Autoscalers" required={FLAGS.MACHINE_AUTOSCALER} />
+      <HrefLink href={formatNamespacedRouteForResource(referenceForModel(MachineModel), 'openshift-machine-api')}
+        name="Machines"
+        required={FLAGS.CLUSTER_API}
+        startsWith={machineStartsWith} />
+      <HrefLink href={formatNamespacedRouteForResource(referenceForModel(MachineSetModel), 'openshift-machine-api')}
+        name="Machine Sets"
+        required={FLAGS.CLUSTER_API}
+        startsWith={machineStartsWith} />
```
Comment (spadgett) [inline frontend/public/components/nav/admin-nav.tsx]: I don't think these items can have the same `startsWith`. Won't they both be highlighted?

**Comment by jhadvig** [comment]
@spadgett found out that the `startsWith` prop doesn't need to be set, to have the  proper item highlighted.
PTAL

**Pull request merged**